### ISSUE TITLE
[bug] laravel9   Class "Illuminate\Support\Facades\Process" not found

### ISSUE THREAD

**Issue by carlin-rj** [stevebauman/autodoc-facades#4]
laravel9 Execution error！Process only runs on laravel10, Can you consider an implementation method that is compatible with lower versions?
```
php artisan autodoc:facades app
```
```
   Error 

  Class "Illuminate\Support\Facades\Process" not found

  at vendor/stevebauman/autodoc-facades/src/Commands/DocumentFacades.php:45
     41▕                 fn (string $classname) => ! in_array($classname, $except)
     42▕             );
     43▕         }
     44▕ 
  ➜  45▕         $result = Process::run(sprintf('php -f vendor/bin/facade.php -- %s', $facades->map(
     46▕             fn (string $classname) => str_replace('\\', '\\\\', $classname)
     47▕         )->join(' ')));
     48▕ 
     49▕         $result->successful()

      +13 vendor frames 
  14  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
```

**Comment by stevebauman** [comment]
Ah forgot this was the case, thanks for the report @carlin-rj! I'll refactor it to use `symfony/process` instead and close this issue once released 👍

**Comment by stevebauman** [comment]
Fixed in [v1.1.2](https://github.com/stevebauman/autodoc-facades/releases/tag/v1.1.2)! Please run `composer update` and you're all set 🙏
